### DISCUSSION TITLE
Default Suggested Zones

### DISCUSSION THREAD

**Discussion by SeanPM5** [home-assistant/frontend#11229] [Other feature requests]
## The request

I think there's a handful of very common zones amongst most users - Home, School, Work, Gym, Supermarket. It might be helpful to show these as suggestions in the Zones Editor sidebar, like this:

![suggested](https://i.imgur.com/ok6XeY9.png)

This gives the user some ideas / inspiration on what zones to add, and gives a better idea of the purpose of zones too (without needing any descriptive text). User looks at that list, and goes, oh those are the places I visit regularly.

These would have the pretty icons already pre-filled, so the user wouldn't need to know about `mdi:dumbbell` for their gym or `mdi:briefcase` for their work etc. The zone name would be pre-filled too - it'd say "Add School" in the sidebar but just "School" when editing it. So the user would only be responsible for setting the proper location for it.

Don't want that zone? Simple, just hit the edit icon and press Delete like normal. Cool part of this, it pretty much guarantees everyone tries the new Zones UI editor. Initial thought may be to delete the suggested "School" zone but while you're doing that you realize how awesome the UI method is and start switching from your `zones.yaml` file  to the UI.

**Comment by zsarnett** [comment]
Should this just be more of a description instead of suggestions? Can we really choose 3 suggestions that would be good for most people

**Comment by SeanPM5** [comment]
Well I do think those are probably the most commonly used Zones. If somebody doesn't want a default suggestion they could delete it in literally two clicks, simple and easy enough.

A bunch of major apps do exactly this, for example here is Google Maps, Apple Maps, Bing Maps:
![image](https://i.imgur.com/20iqDQR.jpg)
![image](https://i.imgur.com/onJn7Nk.jpg)
![image](https://i.imgur.com/mkTFTYb.png)

I think it's a bit nicer than a paragraph of descriptive text.

**Comment by bramkragten** [comment]
But they have suggestion to add, not something default added (with what location?) That had to be removed
